packages/cdktf/lib/terraform-module.ts

```diff
@@ -27,21 +28,25 @@ export abstract class TerraformModule
   public readonly version?: string;
   private _providers?: (TerraformProvider | TerraformModuleProvider)[];
   public dependsOn?: string[];
+  public readonly createAssetsFromLocalModules?: boolean;
 
   constructor(scope: Construct, id: string, options: TerraformModuleOptions) {
     super(scope, id, "module");
 
-    if (options.source.startsWith("./") || options.source.startsWith("../")) {
-      // Create an asset for the local module for better TFC support
-      const asset = new TerraformAsset(scope, `local-module-${id}`, {
-        path: options.source,
-      });
-
-      // Despite being a relative path already, further indicate it as such for Terraform handling
-      this.source = `./${asset.path}`;
-    } else {
-      this.source = options.source;
+    this.source = options.source;
+
+    if (
+      options.createAssetsFromLocalModules === true ||
+      !("createAssetsFromLocalModules" in options)
```

Comment:
Checking is equal to undefined seems slightly nicer, but perhaps this has functional benefits.

Reply:
I chose this because it was more readable and is using the slightly newer `in` construct. I also feel that testing for `undefined` is a bit more nuanced than the hard boolean in this code.

I am happy to change this if you'd like.

Reply:
I agree on the readability, just don't particularly like the hard-coded string. Fine keeping it though.